### PR TITLE
Header

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: assets/css/header.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.site-header {
+  background: url("../img/header.jpg");
+  height: 200px;
+}
+
+.title-header {
+  color: var(--white);
+  text-align: center;
+  padding-top: 32px;
```
Comment (takrami) [inline assets/css/header.css]: Please use variables.

File: assets/css/header.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  font-weight: var(--bold-font);
+}
+
+.text-header {
+  color: var(--white);
+  text-align: center;
+  padding: 16px;
```
Comment (takrami) [inline assets/css/header.css]: Same here.

File: assets/css/header.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+.button-container {
+  display: flex;
+  justify-content: center;
+}
+
+.button-header {
+  background-color: var(--primary-color);
+  color: var(--white);
+  padding: 8px 16px;
```
Comment (takrami) [inline assets/css/header.css]: and here:)

**Review by takrami — APPROVED** [review]
Please resolve the comments then you can merge it :+1:

File: assets/css/header.css (hunk 1 above)
Comment (hedita) [reply assets/css/header.css]: Done.

File: assets/css/header.css (hunk 2 above)
Comment (hedita) [reply assets/css/header.css]: fixed.

File: assets/css/header.css (hunk 3 above)
Comment (hedita) [reply assets/css/header.css]: fixed.

**Pull request merged**